src/components/common/navbar.tsx

```diff
@@ -5,142 +5,258 @@ import Image from 'next/image'
-                                <Typography fontWeight={"bold"} textTransform={"capitalize"}>
-                                    Paintings
-                                 </Typography>
-                            </Button>
-                        </Link>
-                        <Link href='/stories'>
-                            <Button color="primary" className={navigatorStyle} >
-                                <Typography fontWeight={"bold"} textTransform={"capitalize"}>
-                                    Stories
-                                </Typography>
-                            </Button>
-                        </Link>
-                        <Link href='/manuscripts'>
-                            <Button color="primary" className={navigatorStyle}>  
-                                <Typography fontWeight={"bold"} textTransform={"capitalize"}>
-                                    Manuscripts
-                                </Typography>
+                                <Link href="/paintings/manuscript">
+                                    <MenuItem onClick={handleClose} sx={menuStyles}>Paintings by Manuscripts</MenuItem>
+                                </Link>
+                            </Menu>
+
+                            <Link href="/manuscripts">
+                                <Button color="primary">Manuscripts</Button>
+                            </Link>
+
+                            <Button color="primary" onClick={handleClickResearch}>
+                                Research Tools
                             </Button>
-                        </Link>
-                        <Link href='/about'>
-                            <Button color="primary" className={navigatorStyle}> 
-                                <Typography fontWeight={"bold"} textTransform={"capitalize"}>
-                                    About 
-                                </Typography>
+                            <Menu
+                                anchorEl={anchorElResearch}
+                                open={Boolean(anchorElResearch)}
+                                onClose={handleClose}
+                            >
+                                <Link href="/research/incipit-tool">
+                                    <MenuItem onClick={handleClose} sx={menuStyles}>Incipit Tool</MenuItem>
+                                </Link>
+                                <Link href="/research/arabic-manuscripts">
+                                    <MenuItem onClick={handleClose} sx={menuStyles}>Arabic Manuscripts</MenuItem>
+                                </Link>
+                                <Link href="/research/arabic-stories">
+                                    <MenuItem onClick={handleClose} sx={menuStyles}>Arabic Stories</MenuItem>
+                                </Link>
+                                <Link href="/research/maps">
```

Comment:
I think this should go to the page you already created at `/maps` or switch the page you created to be found at `/research/maps`

Reply:
As discussed on our call nevermind